src/js/Draw/L.PM.Draw.Marker.js

```diff
@@ -194,4 +196,62 @@ Draw.Marker = Draw.extend({
       this._hintMarker?.setIcon(this.options.markerStyle.icon);
     }
   },
+  _setupTouchScreenTips() {
+    let markerHintTip = null;
+
+    this._map.on('pm:drawstart', (event) => {
```

Comment:
Eventlistener should be only added if the global option for mobile tooltip is enabled